### PR TITLE
Add "Reclaim all" button

### PR DESCRIPTION
**Pull request by lukaw3d**
Fixes part of https://github.com/oasisprotocol/oasis-wallet-web/issues/492

reclaim all

### CODE REVIEW

**Comment by pro-wh** [comment]
what ranges of numbers does this work on before losing precision? up to ~9M giga shares?

File: src/app/components/ReclaimEscrowForm/index.tsx [hunk 1]
```diff
@@ -49,8 +49,19 @@ export const ReclaimEscrowForm = memo((props: Props) => {
...
+      }),
+    )
+  }
+
+  const reclaimAll = () => {
+    dispatch(
+      transactionActions.reclaimEscrow({
+        type: 'reclaimEscrow',
+        amount: Number(props.maxAmount) / 10 ** 9,
+        shares: Number(props.shares) / 10 ** 9,
```
Comment (lukaw3d) [inline src/app/components/ReclaimEscrowForm/index.tsx:63]: Further refactored in https://github.com/oasisprotocol/oasis-wallet-web/pull/932 to keep precision

**Comment by lukaw3d** [comment]
@pro-wh between `8000000000000001 / 10**9 * 10**9` and `9000000000000001 / 10**9 * 10**9`

**Review by pro-wh — APPROVED** [review]
good for us small stakers imo

^^ the above numbers corresponding to 8-9 million gigashares, which is what you'd get for delegating ~8 million ROSE to a new validator

File: src/app/components/ReclaimEscrowForm/index.tsx (hunk 1 above)
Comment (matevz) [reply src/app/components/ReclaimEscrowForm/index.tsx:63]: FYI shares should be treated as integer too, see #79.

**Comment by matevz** [comment]
Can you also please make "Corresponding number of shares" term translatable in this PR.

File: src/app/components/ReclaimEscrowForm/index.tsx (hunk 1 above)
Comment (pro-wh) [reply src/app/components/ReclaimEscrowForm/index.tsx:63]: too many things are using gigashares lol

**Review by tjanez — APPROVED** [review]

**Pull request merged**